lib/ex_doc/language/elixir.ex

```diff
@@ -109,11 +109,14 @@ defmodule ExDoc.Language.Elixir do
       if actual_def in module_data.private.optional_callbacks, do: ["optional"], else: []
 
     specs =
-      case Map.fetch(module_data.private.callbacks, actual_def) do
-        {:ok, specs} ->
+      case {kind, Map.fetch(module_data.private.callbacks, actual_def)} do
+        {:macrocallback, {:ok, specs}} ->
+          Enum.map(specs, &remove_callback_term/1)
+
+        {:callback, {:ok, specs}} ->
```

Comment:
```suggestion
      case Map.fetch(module_data.private.callbacks, actual_def) do
        if kind == :macrocallback do
          Enum.map(specs, &remove_callback_term/1)
        else
          specs
        end
```

Reply:
@wojtekmach I was also curious about "preferred style" in core team. That was also my first version, but I was thinkining that I'm adding unnecessary indention, so I took a look at other `case` in this file …

I found similar code (variable + function call in tuple) in lines 770 and 835 (or 759 and 824 before change). Before commit those I would like to ask about consistency with mentioned lines if you don't mind. :slightly_smiling_face:

Reply:
José Valim already fixed that much better with `when`. :+1: :heart: